Sources/SwiftDocC/Indexing/Indexable.swift

```diff
@@ -47,7 +47,7 @@ extension Sequence where Element: TextIndexing {
     }
 }
 
-extension Sequence where Element == RenderBlockContent {
```

Comment:
The first example can't be written as `Collection<DocCSymbolRepresentable>` because that would change its meaning from `where Element: DocCSymbolRepresentable` to `where Element == DocCSymbolRepresentable` (which also doesn't compile). In other places this would be described by `Collection<some DocCSymbolRepresentable>` but syntax wasn't supported in extensions.

Reply:
Got it - I see the new generic syntax refers to the associated type of the parent type.

Reply:
Specifically the _primary_ associated type. `Collection` is generic over both the `Element` and `Index` but only `Element` is the _primary_ associated type.